src/main/java/racingCar/Car.java

```diff
@@ -0,0 +1,29 @@
+package racingCar;
+
+import java.util.Random;
+
+public class Car {
+    private static final Random random = new Random();
+    private final int MOVE_MIN_NUM = 4;
```

Comment:
상수로 선언하고자 한다면 관례에 따라 `static` 키워드를 붙여보면 어떨까요?
```suggestion
    private static final int MOVE_MIN_NUM = 4;
```